tests/integration-tests/tests/update/test_update/test_dynamic_file_systems_update/pcluster.config.update_drain.yaml

```diff
@@ -0,0 +1,103 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  Iam:
+    S3Access:
+      - BucketName: {{ bucket_name }}
+Scheduling:
+  SlurmSettings:
+    QueueUpdateStrategy: DRAIN
+  Scheduler: slurm
+  SlurmQueues:
+    - Name: queue1
+      Iam:
+        S3Access:
+          - BucketName: {{ bucket_name }}
+      ComputeResources:
+        - Name: queue1-i1
+          InstanceTypeList:
+            - InstanceType: c5.xlarge
+          MinCount: 1
+          MaxCount: 2
+      Networking:
+        SubnetIds:
+          - {{ private_subnet_id }}
+    - Name: queue2
+      Iam:
+        S3Access:
+          - BucketName: {{ bucket_name }}
+      ComputeResources:
+        - Name: queue2-i1
+          InstanceTypeList:
+            - InstanceType: c5.xlarge
+          MinCount: 1
+          MaxCount: 2
+      Networking:
+        SubnetIds:
+          - {{ private_subnet_id }}
+SharedStorage:
+  - MountDir: /raid_dir
+    StorageType: Ebs
+    Name: raid-type1
```

Comment:
We have both managed and existing file systems. What about prefixing managed file system names with "managed-" as you did for "existing-"?

Reply:
good idea, changed